subprojects/docs/src/docs/userguide/basePlugin.adoc

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[base_plugin]]
+== The Base Plugin
+
+The Base Plugin provides features that are common to most builds and adds a structure to the build that promotes consistency in how they are run. Its most significant contribution is a set of <<sec:base_tasks,'lifecycle' tasks>> that act as an umbrella for the more specific tasks provided by other plugins and build authors.
```

Comment:
Is there a reason why you single-quoted "lifecycle"?

Reply:
- [ ] Italicise 'lifeycle' instead of quoting it

Reply:
Maybe we can directly reference this subsection here: https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:lifecycle_tasks

Reply:
Good idea. Will be in the next commit.